### PR TITLE
chore: Prep for release 4.8.0

### PR DESCRIPTION
**Pull request by zashraf1985**

### CODE REVIEW

File: packages/optimizely-sdk/CHANGELOG.MD [hunk 1]
```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+## [4.8.0] - November 29, 2021
+
+### New Features
+- Added a Lite bundle which does not include Data file manager and Event Processor packages. This reduces the bundle size up to 20% and is helpful for some platforms (such as edge service providers) that do not need extended functionality offered by these packages.
+- Removed Client engine validation in the SDK to allow tracking events from new clients without modifying SDK code.
+
+### Performance Improvements
+- Reduced SDK client initialization time by removing `OptimizelyConfig` creation from initialization. The `OptimizelyConfig` object is now created on the first call to `getOptimizelyConfig` API.
+- Made Improvements to logging mechanism. The SDK now no longer concatenates and formats messages which do not qualify for the log level set by the user.
```
Comment (The-inside-man) [inline packages/optimizely-sdk/CHANGELOG.MD:18]: nit: I would remove the work "now" from "The SDK now no longer...."

File: packages/optimizely-sdk/CHANGELOG.MD [hunk 2]
```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### Changed
+- Updated `json-schema` package version to `0.4.0` to fix a high-severity vulnerability.
```
Comment (The-inside-man) [inline packages/optimizely-sdk/CHANGELOG.MD:21]: Can we give a bit more information on the vulnerability?

**Review by The-inside-man — APPROVED** [review]
Just a couple small comments but LGTM!

File: packages/optimizely-sdk/CHANGELOG.MD (hunk 1 above)
Comment (zashraf1985) [reply packages/optimizely-sdk/CHANGELOG.MD]: Done!

File: packages/optimizely-sdk/CHANGELOG.MD (hunk 2 above)
Comment (zashraf1985) [reply packages/optimizely-sdk/CHANGELOG.MD]: Done! Added a link to the vulnerability.

**Review by The-inside-man — APPROVED** [review]
lgtm

**Pull request merged**